WPMedia/fusion-news-theme-blocks
# TMEDIA-196 - Change imports to utilize the SDK version of constructSocialURL

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description
Removes old social link generating code and utilize the SDK version.

## Jira Ticket
- [TMEDIA-196](https://arcpublishing.atlassian.net/browse/TMEDIA-196)

## Acceptance Criteria
N/A

## Test Steps
- This implements code on the `author-bio-block` and `full-author-bio-block` blocks to handle returning the url for social links on the biography pages.  They will now utilize the shared SDK version of this code which is a near duplicate of the old implementation with some developer QOL improvements.  No visible impact should be noticed on biography pages and the social links should act as they had before.

1. Checkout this branch `git checkout TMEDIA-196-construct-social-media-links`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/author-bio-block,@wpmedia/full-author-bio-block`
3. Find a biography page ([localhost full-author-bio example](http://localhost/pagebuilder/editor/curate?p=p09JD0qqD50JRr&v=v059xCnqD50JRr)) and validate the social media links contained therein.

## Effect Of Changes
### Before
When clicking on the social media links it would direct you to the appropriate URL.

### After
When clicking on the social media links it would direct you to the appropriate URL as before.  No differences should be noted.

## Dependencies or Side Effects
Depends on: https://github.com/WPMedia/engine-theme-sdk/pull/314

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
